learn/why-ballerina/flexibly-typed.md

```diff
@@ -87,15 +87,15 @@ The new type `boolean_false` can be defined in Ballerina code in the following m
 type boolean_false false;
 ```
 
-This example uses the value `false` in defining the new `boolean_false` type. In a more practical scenario, you can provide multiple values as a union when defining new types using the `T1|T2` syntax. A type that only denotes a single shape is called a singleton type. This new type can be used in the code in the following way.
+This example uses the value `false` to define the new `boolean_false` type. In a more practical scenario, you can provide multiple values as a union when defining new types using the `T1|T2` syntax. A type that denotes only a single shape is called a singleton type. This new type can be used in the code in the following way.
 
 ```ballerina
 boolean_false bv1 = false;
 boolean bv2 = true;
 bv2 = bv1;
 ```
 
-As you can see, `bv1` of type `boolean_false` can be assigned to `bv2` of type boolean because `bv1`’s type is a subset of `bv2`’s type. In simple terms, all the values that can be held in the variable `bv1` can be held in the variable `bv2`, thus the assignment is possible.
+As you can see, `bv1` of type `boolean_false` can be assigned to `bv2` of type `boolean` because `bv1`’s type is a subtype of `bv2`’s type. In simple terms, all the values that can be held by the variable `bv1` can be held by the variable `bv2`, thus the assignment is possible.
```

Comment:
```suggestion
As you can see, `bv1` of type `boolean_false` can be assigned to `bv2` of type `boolean` because `bv1`’s type is a subtype of `bv2`’s type. In simple terms, all the values that can be held by the variable `bv1` can be held by the variable `bv2`, and thereby, the assignment is possible.
```